src/pudl/metadata/resources.py

```diff
@@ -198,7 +198,7 @@
         "sources": ["eia923"],
     },
     "generation_fuel_eia923": {
-        "description": "Monthly electricity generation and fuel consumption reported for each combination of fuel and prime mover within a plant.",
+        "description": "Monthly electricity generation and fuel consumption reported for each combination of fuel and prime mover within a plant. This table does not include nuclear fuels because nuclear fuels are reported by generator unit where as non nuclear fuels are reported by plant_id, prime_mover and fuel_type. See the generation_fuel_nuclear_eia923 table for nuclear fuel data.",
```

Comment:
Saying "nuclear fuels" seems a bit odd to me here. It's really electricity generation and fuel consumption.

Reply:
Updated 👍